geniesp/bpc_redcap_export_mapping.py

```diff
@@ -1369,11 +1350,10 @@ def run(self):
                 subset_sampledf[col] = years
 
         # Remove SAMPLE_TYPE and CPT_SEQ_DATE because the values are incorrect
-        del subset_sampledf["SAMPLE_TYPE"]
```

Comment:
Why the removal of SAMPLE_TYPE?

Reply:
We actually don't want to remove it because the value in the subset_sampledf comes from a mapped derived variable in the BPC data files.  Previously, the BPC SAMPLE_TYPE was removed and then replaced by the SAMPLE_TYPE from main GENIE.  To maintain consistency with the BPC curated/derived data, we no longer wish to pull from main GENIE